File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/bindings/NameBinding_ViewModel.cs
/// <filename>
///     NameBinding_ViewModel.cs
/// </filename>
using System.Collections.Generic;
using bindings;

namespace bindings
{
    public class NameBinding_ViewModel
  {
        private string mText;
     
        public     virtual string getCustomText()
    {
      return this.mText;
    }
     
        private bool mEnabled;
     
        public     virtual bool getCustomEnabled()
    {
      return this.mEnabled;
    }
     
        private System.Collections.Generic.List<bindings.MyCustomRow> rows = new System.Collections.Generic.List<bindings.MyCustomRow>();
     
        public     virtual System.Collections.Generic.List<bindings.MyCustomRow> getRows()
    {
      return this.rows;
    }
     
        public     virtual void loadView()
    {
      
    }
     
        public     virtual void OkClicked()
    {
      
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/bindings/NameBinding_ViewModel.cpp

#include "NameBinding_ViewModel.h"
#include "NameBinding_ViewModel.h"
#include <vector>
#include <memory>
#include "MyCustomRow.h"
#include <string>

namespace bindings
{
  std::string NameBinding_ViewModel::getCustomText( ) 
  {
    return this->mText;
  }
  bool NameBinding_ViewModel::getCustomEnabled( ) 
  {
    return this->mEnabled;
  }
  std::vector<std::shared_ptr<MyCustomRow>> NameBinding_ViewModel::getRows( ) 
  {
    return this->rows;
  }
  void NameBinding_ViewModel::loadView( ) 
  {
    
  }
  void NameBinding_ViewModel::OkClicked( ) 
  {
    
  }
}

